transform/transformers/common_software/mbs_transformer.py

```diff
@@ -273,9 +272,7 @@ def transform(self):
 
         return {
             k: v
-            for k, v in merge_dicts(
-                transformed_data, employee_totals, turnover_totals, dates
-            ).items()
+            for k, v in {**transformed_data, **employee_totals, **turnover_totals, **dates}.items()
```

Comment:
Do you need to do the `for k, v...` in this?  The other ones you've changed are more simply `{**dict1, **dict2, **dictN}`.
If not, then could you change this as it'll be much simpler to read.